macos/FluentUI/Button/Button.swift

```diff
@@ -201,11 +204,25 @@ open class Button: NSButton {
 	@objc public var style: ButtonStyle = .primaryFilled {
 		didSet {
 			if style != oldValue {
+				if style == .primaryFilled {
+					contentTintColor = .white
+				} else {
+					contentTintColor = primaryColor
+				}
 				needsDisplay = true
 			}
 		}
 	}
 
+	/// Background color in rest state. Hover and pressed state colors will adjust accordingly using the systemEffect.
+	@objc public lazy var restBackgroundColor: NSColor  = fillColor {
```

Comment:
One thing I wonder about now is the interplay between contentTintColor, primaryColor, and restBackgroundColor as all 3 of those are public and separately configurable, and could get in each other's way?

Reply:
Worth checking! I'd imagine @chiuam can verify those work as expected when toggled separately in the test app?

Reply:
Turns out @chripear is making major changes to the Button properties. Since my changes are a subset of his, maybe it's more time efficient to have him do the thorough testing on this?